src/Miningcore/Blockchain/Ethereum/EthereumPayoutHandler.cs

```diff
@@ -471,10 +481,13 @@ private async Task DetectChainAsync()
             // convert network
             var netVersion = results[0].Response.ToObject<string>();
             var parityChain = isParity ? results[1].Response.ToObject<string>() : (extraPoolConfig?.ChainTypeOverride ?? "Mainnet");
+            var chainIdResult = results[2]?.Response?.ToObject<string>();
+
+            logger.Info(() => $"Ethereum chain ID: {chainId}");
```

Comment:
Do we want log this at WARN?

Reply:
as INFO, we can get it from SumoLogic.  eventually, once we're happy with the functionality, i think we can demote it to DEBUG